tools/clang/unittests/HLSL/DxilModuleTest.cpp

```diff
@@ -514,51 +514,67 @@ TEST_F(DxilModuleTest, SetValidatorVersion) {
     return;
 
   // Current version
-  VerifyValidatorVersionMatches(L"ps_6_2", {});
-  VerifyValidatorVersionMatches(L"lib_6_3", {});
+  VerifyValidatorVersionMatches(L"ps_6_2", {L"-select-validator external"});
+  VerifyValidatorVersionMatches(L"lib_6_3", {L"-select-validator external"});
```

Comment:
I don't understand why we need any of this for any of these tests.  If these tests fail without this added command line option, I think we have a bug.  We should fix that bug, not add this option to the test.